src/EnergyPlus/GroundTemperatureModeling/FiniteDifferenceGroundTemperatureModel.cc

```diff
@@ -188,7 +173,6 @@ void FiniteDiffGroundTempsModel::getWeatherData(EnergyPlusData &state)
     Real64 horizSolarRad_num;
     Real64 airDensity_num;
     Real64 annualAveAirTemp_num;
-    int denominator;
```

Comment:
[src/EnergyPlus/GroundTemperatureModeling/FiniteDifferenceGroundTemperatureModel.cc:191]:(style),[variableScope],The scope of the variable 'denominator' can be reduced.